src/style.css

```diff
@@ -58,6 +58,188 @@
     position: relative;
 }
 
+.middle-segment{
+    background-color: black;
+     height: 130vh; 
```

Comment:
We never give the height of a container to be more than 100vh. Also this middle section shouldn't have fixed height and try sticking auto.